.github/scripts/backlog-cleanup.js

```diff
@@ -0,0 +1,244 @@
+/**
+ * GitHub Action script for managing issue backlog.
+ * 
+ * Behavior:
+ * - Pull Requests are skipped (only opened issues are processed)
+ * - Skips issues with 'to-be-discussed' label.
+ * - Closes issues with label 'awaiting-response' or without assignees, 
+ *   with a standard closure comment.
+ * - Sends a Friendly Reminder comment to assigned issues without 
+ *   exempt labels that have been inactive for 90+ days.
+ * - Avoids sending duplicate Friendly Reminder comments if one was 
+ *   posted within the last 7 days.
+ * - Marks issues labeled 'questions' by adding the 'Move to Discussion' label.
+ *   (Actual migration to Discussions must be handled manually.)
+ */
+
+const dedent = (strings, ...values) => {
+    const raw = typeof strings === 'string' ? [strings] : strings.raw;
+    let result = '';
+    raw.forEach((str, i) => {
+        result += str + (values[i] || '');
+    });
+    const lines = result.split('\n');
+    const minIndent = Math.min(...lines.filter(l => l.trim()).map(l => l.match(/^\s*/)[0].length));
```

Comment:
To avoid using empty lines in `Math.min`
```suggestion
    const lines = result.split('\n');
    if (!lines.some(l => l.trim())) return '';
    const minIndent = Math.min(...lines.filter(l => l.trim()).map(l => l.match(/^\s*/)[0].length));
```